freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/describe/DescribeFreeIpaResponse.java

```diff
@@ -30,6 +30,9 @@
     @ApiModelProperty(value = ModelDescriptions.ENVIRONMENT_CRN, required = true)
     private String environmentCrn;
 
+    @ApiModelProperty(ModelDescriptions.CHILD_ENVIRONMENT_CRN_LIST)
+    private List<String> childEnvironmentCrns;
```

Comment:
childEnvironmentNames ?

Reply:
everything is crn driven, I would stick with it

Reply:
ok then